resolwe_bio/docker_images/latex/packages-manual/README.txt

```diff
@@ -0,0 +1 @@
+This file enables committing `packages-manual` folder.
```

Comment:
Instead of this you could also create a `.placeholder` file.

Reply:
This would require to remove `.*` line in `.gitignore` file. I'm unsure if want to do that...

Reply:
No, you can just unignore `.placeholder` by adding `!.placeholder`.